lib/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -11,6 +11,33 @@ module Helpers
             sections only: %w[all]
           end
 
+          apipie :method, "Returns next integer stored in a hostgroup param named 'sequence_num', starts and increments it by a given amount" do
+            required :hostgroup_name, String, "hostgroup title to store in (e.g. RHEL7/base)"
+            optional :increment, Integer, "integer increment, can be negative too", default: 1
+            optional :name, String, "name of the sequence (prefix for hostgroup parameter)", default: 'default'
+            optional :prefix, Integer, "leading zeroes", default: 1
+            returns String, "empty string when hostgroup not found"
```

Comment:
Maybe "String with incremented value or empty string if hostgroup not found"?

Reply:
Sure